apps/backend/test/e2e/adoptions.e2e-spec.ts

```diff
@@ -122,6 +161,69 @@ describe('AdoptionsController (e2e)', () => {
       .expect([]);
   });
 
+  describe('/GET adoptions/nearMe', () => {
+    const [user1, user2] = countrymen;
+
+    beforeEach(async () => {
+      await authService.signUp(user2);
+
+      await adoptionsService.createAdoption(pet2Mock, user2.email);
+      await adoptionsService.createAdoption(pet1Mock, user1.email);
+    });
+
+    it('returns all adoptions within a radius of 1km', async () => {
+      return request(app.getHttpServer())
+        .get('/api/v1/adoptions/nearMe?radius=1')
+        .auth(authToken, { type: 'bearer' })
+        .expect(200)
+        .expect((res) => {
+          return expect(res.body).toEqual([
+            expect.objectContaining({
+              ...pet2Mock,
```

Comment:
Why only the pet2Mock is being returned? Can we explicitly pass the coordinates in the setup block to make it clear that only pet2 is within this radius?
BTW this library is pretty cool to create factories:
https://github.com/thoughtbot/fishery

Reply:
<img width="593" alt="Screen Shot 2023-05-07 at 10 46 39" src="https://user-images.githubusercontent.com/19699724/236681336-e727481b-f399-44eb-90b8-c02b5dcfa826.png">

Because that's the pet we link to the user on the setup block. Usually, we wouldn't have to create adoptions to test the GET endpoint. However, I haven't found a standard way to start the in-memory mongo with a db dump or something like that. 

<img width="591" alt="Screen Shot 2023-05-07 at 10 51 41" src="https://user-images.githubusercontent.com/19699724/236681603-cdbe59b4-fbbb-46a1-96d8-03614d4530b3.png">

Basically, during the creation step we get the user coordinates and store it in the adoption document. That's why only pet2 is returned.

Please, lemme know if you have any suggestion to make it more clear